docs/core/tools/dotnet-sln.md

```diff
@@ -204,3 +204,7 @@ dotnet sln [<SOLUTION_FILE>] remove [-h|--help]
   The following screenshot shows the result in Visual Studio 2019 **Solution Explorer**:
 
   :::image type="content" source="media/dotnet-sln/dotnet-sln-solution-folder.png" alt-text="Solution Explorer showing class library projects grouped into a solution folder.":::
+
+## See also
+
+- [dotnet/sdk GitHub repo](https://github.com/dotnet/sdk) (dotnet cli source)
```

Comment:
```suggestion
- [dotnet/sdk GitHub repo](https://github.com/dotnet/sdk) (.NET CLI source)
```

Reply:
Is that the current way to refer to the dotnet.exe?

Reply:
Depends on context and what specifically is referred to.  For the file name, *dotnet.exe*; as a tool set, `.NET CLI`